build/build.deps.xml

```diff
@@ -70,13 +74,27 @@
     </sequential>
   </macrodef>
 
+  <!-- Vendor Dependencies for umple.jar -->
+  <target name="deps.vendor" description="Install vendor dependencies">
```

Comment:
This whole target, could we just expand some of the jars? For example, adding a separate "vendor" configuration within the Ivy file. This means that we can avoid the manual unzipping. 

Example code: 
```xml
<echo>Installing Vendors...</echo>
<deps-resolve conf="core" pathid="core.ivy.classpath" />
<foreach item="String" in="${core.ivy.classpath}" delim=":" property="jar.path">
  <unjar src="${jar.path}" dest="${some.staging.path.dir}" />
</foreach>

<jar ...>
  <include>${some.staging.path.dir}</include>
</jar>
```

This is very pseudo- but I just don't like the idea of adding manual management of dependencies when they are already downloaded via Ivy.